build.sbt

```diff
@@ -418,15 +419,26 @@ lazy val bloopgunShaded = project
     fork in Test := true,
     bloopGenerate in Compile := None,
     bloopGenerate in Test := None,
-    libraryDependencies ++= List(Dependencies.scalaXml, Dependencies.scalaCollectionCompat)
```

Comment:
I wonder what scalaXml was need for here? I can see the usage only in backend, which does not explicitly depend on it. Anyway, it looks like it's save to remove.

Reply:
It might be a leftover from past, but it's not needed now.